### ISSUE TITLE
po4a.org down

### ISSUE THREAD

**Issue by jmaibaum** [mquinson/po4a#543]
Hi, I noticed that the website at https://po4a.org is unreachable since about a week now. Our CI/CD at https://github.com/flathub/org.kicad.KiCad` using `flatpak-external-data-checker` fails since then, because we use to query po4a.org to check for new versions of po4a. If the outage takes longer (or maybe the website has been abandoned?), I might consider to switch that check over to looking for new tags on the git repo or similar instead.

**Comment by mquinson** [comment]
Maybe @duck-rh could have an idea about what's going on? I confess that I don't...

**Comment by duck-rh** [comment]
Quack,

I'm sorry about the downtime.
We've been trying to upgrade all VMs out of obsolete OSes and it looks like my colleague zealously went through. I'm not sure what happened exactly but I found the VM blocked on the post-upgrade stage (Leap adds a GRUB entry to finish the upgrade). Unfortunately it failed and then stopped waiting for input.
I was able to get the website up again by using the latest kernel and I'll check with my colleague what would be the next step.

\\_o<
